### ISSUE TITLE
Name is not defined for the combo box under the filter section in Windows.

### ISSUE THREAD

**Issue by msftedad** [telerik/kendo-react#3581]
### Description

**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7670)
Browsers: Edge (Version 145.0.3800.22), Chrome (Version 144.0.7559.110)
Screen readers: NVDA (2025.3.2), JAWS (2026.2512.120), Narrator
URL: https://www.telerik.com/kendo-react-ui/components/grid

**User Impact:**
Users who rely on screen readers may not understand the purpose of the combo box because its accessible name is missing. This can prevent them from selecting the correct filter options, leading to confusion and blocking them from completing filtering tasks independently.

### Plan

KendoReact Free

### Steps To Reproduce

**Pre-Requisite:**

Turn on Screen Readers:

- NVDA: Ctrl + Alt + N
- JAWS: Ctrl + Alt + J
- Narrator: Ctrl + win + Enter

Verbosity:

- NVDA: Default
- JAWS: Beginner, Highest
- Narrator: Default

**Repro Steps:**

1. Open the URL: https://www.telerik.com/kendo-react-ui/components/grid in edge browser in windows.
2. 'React Data Grid Overview' page will open.
3. Press tab key to reach to the 'More Option' button and activate it.
4. Press tab key to reach to the 'Filters' control and activate it.
5. Filters controls will appear.
6. Press tab key to navigate to the combo box present under Filter control.
7. Verify whether name is defined for the combo box under the filter section in Windows or not.

### Screenshots or video

**Refer Attachment:**

1. Name is not defined for the combo box under the filter section in Windows.png
2. Name is not defined for the combo box under the filter section in Windows.mp4

https://github.com/user-attachments/assets/cd264b24-6eb9-4b73-8f0d-63c0f939a44a
<img width="975" height="523" alt="Image" src="https://github.com/user-attachments/assets/d4fd9031-ba8b-4242-af40-ae5a8ffd4fed" />

### Actual Behavior

Name is not defined for the combo box under the filter section in Windows.
Screen reader announces: Combo box contains select collapsed open list contains

### Expected Behavior

Name is not defined for the combo box under the filter section in Windows.
Screen Reader should announce: Operator combo box contains select collapsed open list contains

### Browser

Chrome, Microsoft Edge

### Browser version

Edge (Version 145.0.3800.22), Chrome (Version 144.0.7559.110)

### OS type

Windows

### OS version

Windows 11 version 25H2 (OS Build 26220.7670)

**Comment by PekoPPT** [comment]
HI, @msftedad. 
Thank you for the shared details. I am marking the reported behavior as a bug.

**Comment by msftedad** [comment]
Hi @PekoPPT, Any update on this issue?

**Comment by PekoPPT** [comment]
Hi, @msftedad . The bug is already in our planning for Q4, starting at the end of August. We will do our best to prioritize it before the start of Q4, but I cannot promise an exact date.
